website/src/word-panel.tsx

```diff
@@ -129,6 +126,28 @@ export const WordPanel = (p: {
   }
 }
 
+export const PartLines = (p: {
+  segments: Dailp.FormFieldsFragment["segments"]
+}) => {
+  if (p.segments) {
+    let length = p.segments.length
+    return (
+      <div>
+        {p.segments.map((part, index) => (
+          <tr>
+            <td className={css.table}>
+              {index > 0 ? "-" : null}
+              {part.morpheme}
+              {index === length - 1 ? null : "-"}
+            </td>
+            <td className={css.table}>{part.matchingTag?.learner?.title}</td>
```

Comment:
You can't just use `matchingTag.learner.title` because that's specifically the learner definition. Each tag set / abbreviation system has its own definition for each morpheme tag, so you need to use the one from the currently selected system. Check out the `morphemeDisplayTag` function in `website/src/types.ts` which will give you the right object given `matchingTag` as the first arg and the current tag set as the second arg.